scripts/check-go-vulnerabilities.sh

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
In case you didn't see, this script is working locally for me, but is failing in CI.

Reply:
Yeah... I've been trying to debug it but I'm really not sure why it's giving the error in CI. I tested with the same exact go version and according to the logs it has the error in stderr but then it says "no vulnerabilities found" in stdout. If you have some insight on the tool please lmk! If not no worries, I will continue to debug 🔨

Reply:
I wonder if it is some weird shell thing. Maybe try `set -x` at the top of the script to verify that it's doing what you expect?